lib/decanter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Decanter
-  VERSION = '4.0.1'.freeze
+  VERSION = '4.0.2'.freeze
```

Comment:
Not 100% sure on versioning semantics for minor feature. I suppose this could be `4.1`?

Reply:
This is a backwards compatible feature, which warrants a `minor` bump (i.e., `4.1.0`)